doctest/build.gradle

```diff
@@ -45,7 +45,9 @@ task stopOpenSearch(type: KillProcessTask)
 doctest.dependsOn startOpenSearch
 doctest.finalizedBy stopOpenSearch
 
-build.dependsOn doctest
+// Deactivate doctest in the main gradle workflow (./gradlew build), because doctest depends
+// on ML plugin and it ruins SQL plugin build if ML plugin is not available (not released)
+// build.dependsOn doctest
```

Comment:
if it's for CI only, maybe disable in workflow instead of removing doctest from build task?

But could you explain more on why? If ml-commons is missing and doctest cannot run, we should not merge in PR anyways since we don't know if the PR breaks other doctests

Reply:
> we should not merge in PR anyways

I agree. Unfortunately, `gradle` runs doctests before integration tests and before most of the unit tests. This results in a PR with failed `SQL Java CI` workflow, which crashed on doctests.
The proposed change unblocks UT and IT from doctest. The PR will remain blocked by a failing CI workflow - `SQL Doctest CI`. We will be able to identify that all tests really pass and we are waiting for `ML` release to complete the PR.